src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta": {
+    "releasedOn": "2023.09.13"
+  },
+  "added": [
+    {
+      "what": [
+        "splice function in workflows that can change the contents of an `array` by removing or replacing existing elements of different variable types and/or adding new elements in place."
```

Comment:
Saying `...different variable types...` raises more questions than gives answers -- what types? I can assume but the reader won't know that since it's not some industry standard language.
Please specify all of the types it supports. If it's just arrays, say it's just for arrays.